src/Infrastructure/Persistence/Doctrine/Fixtures/VehicleSetFixture.php

```diff
@@ -17,9 +18,10 @@ public function load(ObjectManager $manager): void
         $vehicleSet1 = new VehicleSet(
             '981f0260-948e-45e9-8788-efa23859a884',
             measure: $this->getReference('measure4'),
-            restrictedTypes: [VehicleTypeEnum::HEAVY_GOODS_VEHICLE->value, VehicleTypeEnum::CRITAIR_5->value],
+            restrictedTypes: [VehicleTypeEnum::HEAVY_GOODS_VEHICLE->value, CritairEnum::CRITAIR->value],
             exemptedTypes: [VehicleTypeEnum::PEDESTRIANS->value, VehicleTypeEnum::AMBULANCE->value, VehicleTypeEnum::OTHER->value],
             otherExemptedTypeText: 'Convois exceptionnels',
+            critairTypes: [CritairEnum::CRITAIR_4->value],
```

Comment:
Est-ce qu'on créerait ici un cas "par défaut", c'est-à-dire 4 et 5 ?

```suggestion
            critairTypes: [CritairEnum::CRITAIR_4->value, CritairEnum::CRITAIR_5->value],
```